src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx

```diff
@@ -366,9 +341,54 @@ The following table shows the default retention settings for our [New Relic One
+
+### User roles and retention
+
+In order to edit retention, a V1 user must be the **Owner** or have the **Data Retention Manager** add-on role. A V2 user must have the **billing_user** role. 
+
+### Reduce retention values
+
+When you reduce the number of days that specific data sources are retained, existing data of that source type won't immediately disappear and might be visible and returned in queries for up to several weeks. We have safeguards in place to ensure you don't accidentally delete data, but this also means that reducing retention values isn't a good strategy for quickly [removing data](#data-deletion).
+
+That said, your change is registered in the system within 12 hours and new data of the adjusted source type will be retained for the new duration. 
+
+<Callout variant="tip">
+Reducing data retention below your contracted retention values will not reduce New Relic’s data ingest charges. Conversely, increasing retention for targeted telemetry data types is likely to trigger a conversation with New Relic about adjusting your data ingest charges.
+  </Callout>
+
+## Copy retention values
+
+If you have permission to edit retention settings, you can also copy retention values from one child account to other child accounts under the same parent account. To do this, from within a child account, select the **Copy settings to other accounts** link, and then select the child accounts you want to apply the retention values to.
+
+![image of child accounts to copy retention values to](./images/copy-data-retention.png "Copy retention values")
+
+<figcaption>
+  Select the child accounts you want to copy retention values to.
+</figcaption>
+
+## Data deletion
+
+Once [telemetry data](/docs/using-new-relic/data/understand-data/new-relic-data-types) (events, metrics, logs, traces) is reported to New Relic and available for querying, that data cannot be edited or deleted. This is a purposeful design decision that optimizes New Relic's speed and performance. Data will expire after its [data retention period](#adjust-retention) ends. If you need to delete data, contact your New Relic account representative. 
+
 ## Metric data rollup
 
-[Metric timeslice data](/docs/using-new-relic/data/understand-data/new-relic-data-types#timeslice-data) is generated by APM (both custom metrics and out-of-the-box metrics), mobile, and browser agents. Metric timeslice data is aggregated first to 1-minute rollups retained for 8 days, then aggregated to 1-hour roll-ups for 90 days, with key metric data in turn being stored indefinitely. We meter on the size of the initial 1-minute rollup, not on any of the subsequent rollups. Read on to learn the key metrics by product.
+[Metric timeslice data](/docs/using-new-relic/data/understand-data/new-relic-data-types#timeslice-data) is generated by APM (both custom metrics and out-of-the-box metrics), mobile, and browser agents. Metric timeslice data is aggregated like this:
```

Comment:
I personally don't like "out-of-the-box" because literally, we don't sell New Relic software on disc, packaged in a box! :-) But that's just me. Maybe consider something like "default metrics" (to differentiate from your custom metrics" or "metrics automatically available in New Relic" or something like that?